web/src/condense.ts

```diff
@@ -18,38 +19,47 @@ This library implements two related, similar concepts:
 
 */
 
-function show_more_link($row) {
+function show_more_link($row: JQuery): void {
```

Comment:
You can type these more precisely with `JQuery<foo>`; use `git grep JQuery` to find many examples in the codebase.

Reply:
That’s rarely necessary. The default is `JQuery<HTMLElement>` which is fine here.

Reply:
@timabbott and @andersk thanks for the review. I will adjust the PR accordingly.